Programing-Paradigm-And-Methodology.md

```diff
@@ -0,0 +1,18 @@
+# Programing Paradigm and Methodology
+## Foundational Agile Principles
+**_Individuals and interactions_** over processes and tools
```

Comment:
- reformat using triple asterisks to bold italic
- bold italic like the following

```md
***Individuals and interactions*** over ***processes and tools***
```
> ***Individuals and interactions*** over ***processes and tools***